Makefile

```diff
@@ -207,46 +207,56 @@ endif
 container-runtime-login: ## Log in into the Docker repository
 	@echo "+ $@"
 
-.PHONY: container-runtime-build
-container-runtime-build: check-env deepcopy-gen ## Build the container
+.PHONY: container-runtime-build-%
+container-runtime-build-%: ## Build the container
 	@echo "+ $@"
-	$(CONTAINER_RUNTIME_COMMAND) build \
-	--build-arg GO_VERSION=$(GO_VERSION) \
-	--build-arg CTIMEVAR="$(CTIMEVAR)" \
-	-t $(DOCKER_REGISTRY):$(GITCOMMIT) . \
-	--file Dockerfile $(CONTAINER_RUNTIME_EXTRA_ARGS)
+	$(CONTAINER_RUNTIME_COMMAND) buildx build \
+		--output=type=docker --platform linux/$* \
+		--build-arg GO_VERSION=$(GO_VERSION) \
+		--build-arg CTIMEVAR="$(CTIMEVAR)" \
+		--tag $(DOCKER_REGISTRY):$(GITCOMMIT) . \
```

Comment:
you should add the plateform to the tag name, something like
--tag $(DOCKER_REGISTRY):$(GITCOMMIT)-$* . \